samples/rtl/axpy.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2019-2022 ETH Zurich and the DaCe authors. All rights reserved.
+#
+# This sample shows the AXPY BLAS routine. It is implemented through Xilinx IPs in order to utilize floating point
+# operations. It is intended for running hardware_emulation or hardware xilinx targets.
+
+import dace
+import numpy as np
+
+# add symbol
+N = dace.symbol('N')
+
+# add sdfg
+sdfg = dace.SDFG('axpy')
+
+# add state
+state = sdfg.add_state('device_state')
+
+# add parametr
+veclen = 1
+sdfg.add_constant('VECLEN', veclen)
+sdfg.add_constant('DATA_WIDTH', 64)
+sdfg.add_constant('WORD_WIDTH', 32)
```

Comment:
What are these used for?

Reply:
They were old constants used in a previous version of the samples. They aren't used so they have been removed.